python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cugraph.structure.utils_wrapper import *
+from cugraph.opg.link_analysis cimport mg_pagerank as c_pagerank
+import cudf
+from cugraph.structure.graph_new cimport *
+import cugraph.structure.graph_new_wrapper as graph_new_wrapper
+from libc.stdint cimport uintptr_t
+from cython.operator cimport dereference as deref
+
+def mg_pagerank(input_df, local_data, handle, alpha=0.85, max_iter=100, tol=1.0e-5):
```

Comment:
We don't support `tol` currently, we should throw an error if users set it (or add it later to the API).

Reply:
I think we could add it to the final PR after #947 is merged, along with opg pagerank tests.